homeassistant/components/icloud/sensor.py

```diff
@@ -0,0 +1,106 @@
+"""Battery state for iCloud devices."""
```

Comment:
Please don't add new platforms in this PR. Let's do that in subsequent PRs.

Reply:
Actually, this PR was originally made to add this platform, but, config flow first !